qiime/core/archiver.py

```diff
@@ -66,12 +66,68 @@ def peek(cls, filepath):
 
     @classmethod
     def load(cls, filepath):
-        metadata = cls.peek(filepath)
-        return cls(*metadata, archive_filepath=filepath)
+        def extract_data(data_dir):
+            with zipfile.ZipFile(filepath, mode='r') as zf:
+                root_dir = cls._get_root_dir(zf)
+                prefix = os.path.join(root_dir, cls.DATA_DIRNAME, '')
```

Comment:
Create class-level variable for zip separator (`/`) since `zf.namelist()` will return `/` paths.